### PR TITLE
Expose safe env variables only by /get_config

### PR DESCRIPTION
**Pull request by platfowner**

### CODE REVIEW

File: common/constants.js [hunk 1]
```diff
@@ -652,6 +652,32 @@ const APP_DEPENDENT_SERVICE_TYPES = [
...
+  'HOSTING_ENV',
+  'DEBUG',
+  'PORT',
+  'STAKE',
+  'SYNC_MODE',
+];
+
+function getEnvVariables() {
+  const envs = {};
+  for (const envName of Object.keys(NodeParams)) {
```
Comment (liayoo) [inline common/constants.js:666]: question) is NodeParams used instead of NodeConfigs in order to get more "raw" values?

**Review by liayoo — APPROVED** [review]
LGTM with a question, thanks!

File: common/constants.js (hunk 1 above)
Comment (platfowner) [reply common/constants.js:666]: currently, we're showing such env variables 1) overwriting node params or 2) included in EXPOSED_ENV_VARIABLE_LIST. so if there are env variables not in 1) case, it needs to be added to EXPOSED_ENV_VARIABLE_LIST manually. (if an env variable is in node configs but not in node params, the value is not used in overwriting node params so it should go to the list.)

**Comment by platfowner** [comment]
Thanks for the fast review, let me merge it asap.

**Pull request merged**